### PR TITLE
Adding a welcome screen

### PR DESCRIPTION
**Pull request by sgrieve**
Still a work in progress, the welcome screen is not very aesthetically pleasing. But meets the minimum requirements for #230.

### CODE REVIEW

File: python/nammu/controller/NammuController.py [hunk 1]
```diff
@@ -107,6 +108,14 @@ def __init__(self):
         # Find windows shouldn't coexist
         self.finding = False
 
+        # Here are the current urls for nammu on github and the oracc docs
+        self.urls = {'nammu': 'https://github.com/oracc/nammu',
```
Comment (raquel-ucl) [inline python/nammu/controller/NammuController.py]: I'd be more inclined to move this to WelcomeController. Member variables in Nammu controller should be more things that need to be shared between different controllers, or be strictly related to NammuController, and the url list is only relevant for the welcome window.

File: python/nammu/controller/NammuController.py [hunk 2]
```diff
@@ -645,6 +654,13 @@ def process_validation_errors(self, oracc_log):
         # Refresh validation errors
         self.atfAreaController.set_validation_errors(validation_errors)
 
+    def launchWelcomeScreen(self):
+        '''
+        Checks if new_user flag is true, launches the welcome screen if needed
+        '''
+        if self.config['new_user']:
```
Comment (raquel-ucl) [inline python/nammu/controller/NammuController.py]: ... which just happened in my laptop now :) 

```Exception in thread "MainThread" Traceback (most recent call last):
  File "<string>", line 3, in <module>
  File "<string>", line 3, in <module>
  File "__pyclasspath__/nammu/main.py", line 27, in main
  File "__pyclasspath__/nammu/controller/NammuController.py", line 117, in __init__
  File "__pyclasspath__/nammu/controller/NammuController.py", line 661, in launchWelcomeScreen
KeyError: 'new_user'```

File: resources/config/settings.yaml [hunk 3]
```diff
@@ -105,3 +105,5 @@ servers:
 
 console_style:
   fontsize: 11
+
+new_user: True
```
Comment (sgrieve) [inline resources/config/settings.yaml]: Done in ff11742

File: python/nammu/controller/NammuController.py (hunk 2 above)
Comment (sgrieve) [inline python/nammu/controller/NammuController.py]: Fixed in 49b15c2.

File: python/nammu/controller/NammuController.py (hunk 1 above)
Comment (sgrieve) [inline python/nammu/controller/NammuController.py]: Leaving as is.

**Pull request merged**